packages/actor-init-sparql/lib/ActorInitSparql-browser.ts

```diff
@@ -117,6 +117,10 @@ export class ActorInitSparql extends ActorInit implements IActorInitSparqlArgs {
       context[KEY_CONTEXT_LOG] = this.logger;
     }
 
+    if (!context[KEY_CONTEXT_QUERY_TIMESTAMP]) {
+      context[KEY_CONTEXT_QUERY_TIMESTAMP] = new Date(Date.now());
```

Comment:
This should do the same thing:
```suggestion
      context[KEY_CONTEXT_QUERY_TIMESTAMP] = new Date();
```